cmake/git_watcher.cmake

```diff
@@ -25,7 +25,7 @@
 #     consider that it can run in one of two scopes.
 
 if(NOT DEFINED post_configure_file)
-    set(post_configure_file "${CMAKE_CURRENT_SOURCE_DIR}/src/benchmarklib/git.hpp")
+    set(post_configure_file "${CMAKE_CURRENT_SOURCE_DIR}/src/lib/version.hpp")
```

Comment:
I don't think that this is what @mrzzzrm meant. If I understand him correctly, he was thinking about `./build/version.hpp`.

But wouldn't it be easier to simply add it to .gitignore?

Reply:
it is already in .gitignore

>./build/version.hpp
How would I tell Cmake to include it as a source?

Reply:
No idea. Not even sure if it's a good idea.

If it is in .gitignore, why am I seeing it in the diff?

Reply:
because i renamed the file and forgot to change it in the gitignore

Reply:
```
add_executable(target <other_sources> ${CMAKE_CURRENT_BINARY_DIR}/version.hpp)
include_directories( ${CMAKE_CURRENT_BINARY_DIR}/)
```

my bet would be that the general consensus among projects doing this is to put generated source files into the build dir.

Reply:
I don't really have an opinion about this. If you think the build dir would be better, could you please help with the necessary cmake changes?